pkgs/modules/default.nix

```diff
@@ -60,6 +60,7 @@ let
     (import ./dart)
     (import ./deno)
     (import ./docker)
+    (import ./docker-compose)
```

Comment:
https://github.com/replit/nixmodules/blob/4d6f7cd9fd685a3319ac7b6e3fc0789b430d6289/pkgs/modules/docker/default.nix#L182 docker-compose is already part of the docker module. I think we don't need to add another module for this.